tests/unit/test_configuration.py

```diff
@@ -34,6 +30,68 @@ def test_get_default_configuration(mocker):
     mock.assert_called_once_with(layers=["baseline", "default", "user"])
 
 
+def test_default_configuration_structure():
+    """Test that the default configuration structure is correct"""
+    config = get_configuration()
+    # Check forms
+    assert set(f.name for f in FORMS) == set(config.keys())
+    for form in FORMS:
+        # Check row noise
+        for row_noise in form.row_noise_types:
+            config_probability = config[form.name].row_noise[row_noise.name].probability
+            default_probability = (
+                DEFAULT_NOISE_VALUES.get(form.name, {})
+                .get("row_noise", {})
+                .get(row_noise.name, {})
+                .get("probability", {})
+            )
+            if default_probability:
+                assert config_probability == default_probability
+            else:
+                assert config_probability == getattr(NOISE_TYPES, row_noise.name).probability
+        for col in form.columns:
+            for noise_type in col.noise_types:
+                config_level = config[form.name].column_noise[col.name][noise_type.name]
+                builder_level = getattr(NOISE_TYPES, noise_type.name)
```

Comment:
What does `builder_level` mean?

Reply:
lol I forgot I never went back and changed that name. I had a brain fart and could't think of what to call it besides the configuration "builder". But really they're the baseline values since they're coming out of NOISE_TYPES